src/Docker.PowerShell/Cmdlets/InvokeContainerImage.cs

```diff
@@ -77,7 +78,7 @@ public class InvokeContainerImage : CreateContainerCmdlet
                     }
                     else if (PassThru.ToBool())
                     {
-                        WriteObject(await ContainerOperations.GetContainerById(createResult.ID, DkrClient));
+                        WriteObject((await ContainerOperations.GetContainersById(createResult.ID, DkrClient)).Single());
```

Comment:
That's a good point... I'll work on something for this as part of https://github.com/Microsoft/Docker-PowerShell/issues/20 rather than fix it as part of this change, since it's consistent with what we were doing before.